### DISCUSSION TITLE
What is the "Issues" icon supposed to represent?

### DISCUSSION THREAD

**Discussion by ParkerM** [community/community#76516] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I was unable to find the "Issues" link (that goes to https://github.com/issues) for quite a while after switching to the new UI, until recently I realized it's the little circle with a dot in the middle.

![image](https://github.com/community/community/assets/5124113/7dbdac5e-aeb3-42ac-a3c5-f9245c74c298)

Why is this icon used for the Issues link, or what is the significance behind its design? It seems less intuitive compared to the icons for Pull requests and Notifications, but perhaps I'm missing something obvious.

**Comment by deeptejd** [comment]
It is supposed to represent "Pinpointing an issue" the dot being the issue that is being pinpointed, or rather  a "target"
